packages/obonode/obojobo-chunks-figure/image-properties-modal.js

```diff
@@ -59,15 +81,46 @@ class ImageProperties extends React.Component {
 			>
 				<div className="image-properties">
 					<label htmlFor="obojobo-draft--chunks--figure--url">URL:</label>
-					<input
-						type="text"
-						id="obojobo-draft--chunks--figure--url"
-						value={this.state.url || ''}
-						onChange={this.handleURLTextChange.bind(this)}
-						ref={this.inputRef}
-						size="50"
-						placeholder="Web Address of the Image"
-					/>
+					<div className="flex-container">
+						<div>
+							<input
+								type="text"
+								id="obojobo-draft--chunks--figure--url"
+								value={this.state.urlInputText || ''}
+								onChange={this.handleURLTextChange.bind(this)}
+								ref={this.inputRef}
+								size="50"
+								placeholder="Web Address of the Image"
+							/>
+						</div>
+						<div id="flex-item-2">Or</div>
+						<div id="flex-item-3">
+							<label htmlFor="image-file-input">
+								<input
+									type="file"
```

Comment:
Would be useful to restrict this on filetype - if I try to upload a unsupported file an error shows up in devtools but nothing happens in the UI.

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept